.github/autosubmit.yml

```diff
@@ -0,0 +1 @@
+config_path: 'autosubmit/flutter/autosubmit_main.yml'
```

Comment:
Maybe a comment on how this file will be added? also please add a copyright header.